rowhammer_tester/scripts/rowhammer.py

```diff
@@ -299,8 +332,16 @@ def main(row_hammer_cls):
         data_inversion=args.data_inversion,
     )
 
+    if args.log_dir:
+        log_dir = Path(args.log_dir)
+        if not log_dir.is_dir():
+            log_dir.mkdir(parents=True)
+    row_hammer.log_directory = args.log_dir
```

Comment:
`args.log_dir` is `None` when not given any value by the user, so in this case `row_hammer.log_directory` will be `None` too. But as `row_hammer.log_directory` is defined earlier as `None`, it could be in fact moved to the body of the if condition, so we don't set it to `None` again.